bciers/apps/reporting/src/app/components/activities/ActivityForm.tsx

```diff
@@ -11,6 +10,10 @@ import { FuelFields } from "./customFields/FuelFieldComponent";
 import { FieldProps } from "@rjsf/utils";
 import { getUiSchema } from "./uiSchemas/schemaMaps";
 import { UUID } from "crypto";
+import { withTheme } from "@rjsf/core";
+import formTheme from "@bciers/components/form/theme/defaultTheme";
+
+const Form = withTheme(formTheme);
```

Comment:
`FormBase` was introducing some unintended behaviours by also holding a copy of the formData in the state, so this uses the RJSF component directly